stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Message.kt

```diff
@@ -94,6 +94,14 @@ public data class Message(
     var replyTo: Message? = null,
 
     @SerializedName("quoted_message_id")
-    var replyMessageId: String? = null
+    var replyMessageId: String? = null,
+
+    var pinned: Boolean = false,
+    @SerializedName("pinned_at")
+    var pinnedAt: Date? = null,
+    @SerializedName("pin_expires")
+    var pinExpires: Date? = null,
+    @SerializedName("pinned_by")
+    var pinnedBy: User? = null,
```

Comment:
I assume we should also update `MessageEntity` and `ChannelEntity`?

Reply:
As far as I know, for now, we are going to only support this in LLC so I'm not sure if there is a need to update those entities

Reply:
Ah, okay, I didn't realize the scope of the task. The PR's title is actually clear, the Jira task perhaps not so much.